lib/parseMigrate.js

```diff
@@ -9,32 +9,40 @@ const promisifyWriteFile = promisify(fs.writeFile);
 
 module.exports = async filePath => {
   try {
-    const { stdout: docgenOut } = await promisifyExec(`react-docgen ${filePath} --pretty`);
+    const { stdout: docgenOut } = await promisifyExec(
+      `react-docgen ${filePath} --pretty --resolver findAllComponentDefinitions`
+    );
 
     // Success!
-    const object = await parseProps(JSON.parse(docgenOut));
-
+    const docgenOutArray = JSON.parse(docgenOut);
     console.log('🎉 completed: parsed React component', filePath);
 
-    const { stdout: ctfMigrateOut } = await promisifyExec(
-      `ctf-migrate create ${object.content_type} -c ${object.content_type}`
-    );
+    // Loop over array of components and create migration for each
+    docgenOutArray.forEach(async docgenOut => {
+      const object = await parseProps(docgenOut);
 
-    // Success!
-    const fileName = ctfMigrateOut.split('/')[ctfMigrateOut.split('/').length - 1].trim();
+      if (!object) return;
```

Comment:
When would this happen? This would exit sliently, no? Should we output some messaging to the user if this happens? Is there test for this scenario?

Reply:
take a look at the scenarios here: https://github.com/wework/monarch/pull/26/files/15d0c451c243657ff01c79057e68e85c3de594b2#diff-6b16781044833cf57a27267388bca3e4R7

yes, there are tests here: https://github.com/wework/monarch/pull/26/files/15d0c451c243657ff01c79057e68e85c3de594b2#diff-18743c38e5da203aaa38b7129146e055R421

It seems like the cases are both pretty obvious to ignore, but I can definitely log a message, not too noisy at the moment

Reply:
Might be good to have some messaging to the user.

Reply:
updated:
```
🎉 completed: parsed React component example/components/CrossSellGroup.jsx
ignored component: OtherComponentWithoutPropTypes because it is without proptypes defined
ignored component: OtherComponentToBeIgnored because it is using @ignore-content-type flag
🎉 completed: created migration 20181106171539-CrossSellGroup.js
All done!
```